.github/workflows/autorelease.yml

```diff
@@ -39,6 +39,11 @@ jobs:
           ref: ${{ github.base_ref }}
           fetch-depth: 0
 
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version-file: '.nvmrc'
+
```

Comment:
From: 
https://github.com/actions/setup-node/issues/32

Reply:
It looks right. I'm not sure how I could test it locally.